plugins/module_utils/omevv_utils/omevv_info_utils.py

```diff
@@ -153,3 +154,37 @@ def get_firmware_drift_info_for_multiple_cluster(self, uuid, groupidlist):
                                                                      groupid=each_group)
             result.append(output)
         return result
+
+    def get_cluster_name(self, uuid, host_id):
+        uri = MANAGED_HOST_INFO_URI.format(uuid=uuid)
+        resp = self.omevv_obj.invoke_request('GET', uri)
+        managed_hosts = resp.json_data
+        for host in managed_hosts:
+            if host['id'] == host_id:
+                return host['clusterName']
+        return None
+
+    def get_host_id(self, uuid, hostname=None, servicetag=None):
+        uri = MANAGED_HOST_INFO_URI.format(uuid=uuid)
+        resp = self.omevv_obj.invoke_request('GET', uri)
+        managed_hosts = resp.json_data
+        for host in managed_hosts:
+            if hostname and host['hostName'] == hostname:
+                return host['id'], host['serviceTag']
+            if servicetag and host['serviceTag'] == servicetag:
+                return host['id'], host['serviceTag']
+        return None
```

Comment:
No need to return None explicitly, function default return value is None, when function is not returning anything, applies to other places too

Reply:
This condition is needed. So, not making any changes